routing/probability_estimator.go

```diff
@@ -0,0 +1,52 @@
+package routing
+
+import (
+	"time"
+
+	"github.com/btcsuite/btcd/btcutil"
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/lightningnetwork/lnd/routing/route"
+)
+
+// Estimator estimates the probability to reach a node.
+type Estimator interface {
```

Comment:
Should this just be a private interface given all the methods aren't exported? 

I think exporting the methods can make sense as a public interface, since then it would be possible to pass in ones _own_ estimator at runtime when creating lnd.